freqtrade/configuration/cli_options.py

```diff
@@ -191,6 +191,12 @@ def __init__(self, *args, **kwargs):
         action='store_true',
         default=False,
     ),
+    "print_colorized": Arg(
+        '--color', '--print-colorized',
+        help='Print colorized hyperopt results.',
+        action='store_true',
+        default=False
```

Comment:
I think this provides quite good benefits and improves usability and output readability enormously...

shouldn't this be the default and `--no-color` can disable colorized output (for example if piping the result to a file or so)?

Reply:
Yeah, it improves readability alot, that's why I implemented it... 😆 

No problem, I could make it true by default, but this is much useful with `--print-all` (which I use frequently), which is not a default mode (and should not be default). In the no-print-all mode it just prints profitable results in bold (currently, if colorization schema won't be changed). If you think this is also useful, please confirm and I'll change colorization to be used by default...

Reply:
i would make it default - yes :+1:

Reply:
done.